### ISSUE TITLE
spatial softmax

### ISSUE THREAD

**Issue by ahundt** [jostosh/deeprl#1]
I came across this and thought you may be interested in this relevant tf issue: https://github.com/tensorflow/tensorflow/issues/6271

**Comment by jostosh** [comment]
Thanks for pointing that out @ahundt! I will have a look at it
